conans/client/conf/__init__.py

```diff
@@ -8,181 +8,201 @@
+    # retry_wait = 5                        # environment CONAN_RETRY_WAIT (seconds)
+    # sysrequires_mode = enabled          # environment CONAN_SYSREQUIRES_MODE (allowed modes enabled/verify/disabled)
+    # vs_installation_preference = Enterprise, Professional, Community, BuildTools # environment CONAN_VS_INSTALLATION_PREFERENCE
+    # verbose_traceback = False           # environment CONAN_VERBOSE_TRACEBACK
+    # error_on_override = False           # environment CONAN_ERROR_ON_OVERRIDE
+    # bash_path = ""                      # environment CONAN_BASH_PATH (only windows)
+    # read_only_cache = True              # environment CONAN_READ_ONLY_CACHE
+    # cache_no_locks = True               # environment CONAN_CACHE_NO_LOCKS
+    # user_home_short = your_path         # environment CONAN_USER_HOME_SHORT
+    # use_always_short_paths = False      # environment CONAN_USE_ALWAYS_SHORT_PATHS
+    # skip_vs_projects_upgrade = False    # environment CONAN_SKIP_VS_PROJECTS_UPGRADE
+    # non_interactive = False             # environment CONAN_NON_INTERACTIVE
+    # skip_broken_symlinks_check = False  # environment CONAN_SKIP_BROKEN_SYMLINKS_CHECK
+    
+    # conan_make_program = make           # environment CONAN_MAKE_PROGRAM (overrides the make program used in AutoToolsBuildEnvironment.make)
+    # conan_cmake_program = cmake         # environment CONAN_CMAKE_PROGRAM (overrides the make program used in CMake.cmake_program)
+    
+    # cmake_generator                     # environment CONAN_CMAKE_GENERATOR
+    # cmake generator platform            # environment CONAN_CMAKE_GENERATOR_PLATFORM
+    # http://www.vtk.org/Wiki/CMake_Cross_Compiling
+    # cmake_toolchain_file                # environment CONAN_CMAKE_TOOLCHAIN_FILE
+    # cmake_system_name                   # environment CONAN_CMAKE_SYSTEM_NAME
+    # cmake_system_version                # environment CONAN_CMAKE_SYSTEM_VERSION
+    # cmake_system_processor              # environment CONAN_CMAKE_SYSTEM_PROCESSOR
+    # cmake_find_root_path                # environment CONAN_CMAKE_FIND_ROOT_PATH
+    # cmake_find_root_path_mode_program   # environment CONAN_CMAKE_FIND_ROOT_PATH_MODE_PROGRAM
+    # cmake_find_root_path_mode_library   # environment CONAN_CMAKE_FIND_ROOT_PATH_MODE_LIBRARY
+    # cmake_find_root_path_mode_include   # environment CONAN_CMAKE_FIND_ROOT_PATH_MODE_INCLUDE
+    
+    # msbuild_verbosity = minimal         # environment CONAN_MSBUILD_VERBOSITY
+    
+    # cpu_count = 1             # environment CONAN_CPU_COUNT
+    
+    # Change the default location for building test packages to a temporary folder
+    # which is deleted after the test.
+    # temp_test_folder = True             # environment CONAN_TEMP_TEST_FOLDER
+    
+    # cacert_path                         # environment CONAN_CACERT_PATH
+    # scm_to_conandata                    # environment CONAN_SCM_TO_CONANDATA
+    {% if conan_v2 %}
+    revisions_enabled = 1
```

Comment:
This is not enough, we need to actually check the status when the user already has a previous settings.yml or conan.conf with different values of revisions and cppstd.